Examples/UIExplorer/js/ActivityIndicatorExample.js

```diff
@@ -151,12 +151,23 @@ exports.examples = [
     }
   },
   {
-    title: 'Custom size',
+    title: 'Custom size (size: 39)',
     render() {
       return (
         <ActivityIndicator
-          style={[styles.centering, {transform: [{scale: 1.5}]}]}
-          size="large"
+          style={styles.centering}
+          size={39}
+        />
+      );
+    }
+  },
+  {
+    title: 'Custom size (size: 57)',
+    render() {
+      return (
+        <ActivityIndicator
+          style={styles.centering}
```

Comment:
@fadils Seems Flow is confused regarding the types. Can you add annotation to `getDefaultProps` so that flow doesn't try to infer the type and see if it fixes the issue?